sdc/io/csv_ext.py

```diff
@@ -394,8 +394,8 @@ def to_varname(string):
 
 
 def to_varname_without_excess_underscores(string):
-    """Replaces unavailable symbols with underscores.
-     Are removing excess underscores from the string."""
+    """Using to_varname.
```

Comment:
I think the first part of the docstring is excess. I can see that `to_varname` is used in the code.